lisa/messages.py

```diff
@@ -301,6 +301,24 @@ class ProvisionBootTimeMessage(MessageBase):
     information: Dict[str, str] = field(default_factory=dict)
 
 
+@dataclass
+class UnifiedProvisionBootTimeMessage(PerfMessage):
```

Comment:
Do not create any new message in the messages. The PR is to ask to send `UnifiedPerfMessage` notifications for the boot time message like other places in the code.